src/components/playback/playbackmanager.js

```diff
@@ -1945,8 +1945,8 @@ class PlaybackManager {
                 promise = new Promise(function (resolve, reject) {
                     const apiClient = ServerConnections.getApiClient(firstItem.ServerId);
 
-                    apiClient.getCurrentUser().then(function (user) {
-                        if (!user.Configuration.EnableNextEpisodeAutoPlay || !firstItem.SeriesId) {
+                    apiClient.getCurrentUser().then(function () {
```

Comment:
I think there is no need for `getCurrentUser` now (called to retrieve the user's configuration/policy.)

Reply:
Yeah you're right. Just followed the pattern I saw used elsewhere. It has been updated.

Reply:
You removed wrong `getCurrentUser`.

Reply:
Whoops I misunderstood. Think it's actually fixed this time.